dockers/skylab/HCA_post_processing/optimus_HCA_loom_merge.py

```diff
@@ -2,6 +2,7 @@
 import loompy
 import numpy as np
 
+
```

Comment:
Do we want to remove this space?

Reply:
my linter insists on 2 blank lines before function definitions